contrib/gcppubsub/pkg/controller/channel/reconcile.go

```diff
@@ -22,10 +22,10 @@ import (
 
 	ccpcontroller "github.com/knative/eventing/contrib/gcppubsub/pkg/controller/clusterchannelprovisioner"
 	pubsubutil "github.com/knative/eventing/contrib/gcppubsub/pkg/util"
-	"github.com/knative/eventing/contrib/gcppubsub/pkg/util/logging"
 	eventingv1alpha1 "github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
 	util "github.com/knative/eventing/pkg/provisioners"
 	"github.com/knative/eventing/pkg/reconciler/names"
+	"github.com/knative/eventing/pkg/utils/logging"
```

Comment:
Can this just be `knative/eventing/pkg/logging`? What's the value of the `utils` directory?

Reply:
Done.